docs/controls/RichSuggestBox.md

```diff
@@ -0,0 +1,120 @@
+---
+title: RichSuggestBox
+author: huynhsontung
+description: A rich text input control that auto-suggests and stores token items in a document.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, RichSuggestBox
+dev_langs:
+  - csharp
+---
+
+# RichSuggestBox
+
+The [RichSuggestBox](/dotnet/api/microsoft.toolkit.uwp.ui.controls.richsuggestbox) is a combination of [AutoSuggestBox](/uwp/api/windows.ui.xaml.controls.autosuggestbox) and [RichEditBox](/uwp/api/windows.ui.xaml.controls.richeditbox) that can provide suggestions based on customizable prefixes. Selected suggestions are embedded and tracked in the document as tokens.
+
+RichSuggestBox resembles text controls commonly found in social applications where you type "@" to mention people.
+
+> **Platform APIs:** [`RichSuggestBox`](/dotnet/api/microsoft.toolkit.uwp.ui.controls.richsuggestbox)
+
+> [!div class="nextstepaction"]
+> [Try it in the sample app](uwpct://Controls?sample=RichSuggestBox)
+
+## Syntax
+
+```xaml
+<controls:RichSuggestBox
+  PlaceholderText="Leave a comment"
+  ItemTemplate="{StaticResource SuggestionTemplate}"
+  Prefixes="@#" />
+```
+
+## Example Output
+
+![RichSuggestBox Example](../resources/images/Controls/RichSuggestBox.gif)
+
+## Remarks
+
+When a suggestion is selected, `RichSuggestBox` assigns the selected item a unique [Guid](/dotnet/api/system.guid) and a display text (provided by the developer) to make up a token. The display text is then padded with [Zero Width Space](https://unicode-table.com/200B/)s (ZWSP) and inserted into the document as a hyperlink using the identifier as the link address. These hyperlinks are tracked and validated on every text change.
+
+The token text inserted into the document has the following layout: ZWSP - Prefix character - Display text - ZWSP.
+
+For example, a token with "@" as the prefix and "John Doe" as the display text is inserted as:
+```cs
```

Comment:
This code fragment appeared in-line with the text before it, and didn't appear to be outputting what would be expected.  I put in a blank line before the `'''cs`, and here's the output:  
![image](https://user-images.githubusercontent.com/86266896/133838112-d154d35e-d9b2-47fe-9a8b-f5a0cac05a19.png)

Reply:
I have added a blank line before the code block.